elements/lisk-chain/test/utils/block.ts

```diff
@@ -76,7 +77,7 @@ export const createValidDefaultBlock = async (
 		timestamp: genesis.header.timestamp + 10,
 		transactionRoot: txTree.root,
 		stateRoot: getRandomBytes(32),
-		assetsRoot: getRandomBytes(32),
+		assetsRoot: await blockAssets.getRoot(),
```

Comment:
Prefer to keep the `await` outside of the assignment.
Also, probably we can update `getRoot` to be synchronous function using `calculateMerkleRootWithLeaves`. If it affects too many places, we can split it in another issue

Reply:
Let's do it in a separate issue since it wasn't part of the original estimation